### DISCUSSION TITLE
Analysis of historicals 092 and 098b

### DISCUSSION THREAD

**Discussion by islasimpson** [NCAR/amwg_dev#561] [Discussion about coupled runs]
The CESM2 historical simulations are warming too much in the 20th century.  This seems like it might be related to a lack of influence of aerosols on shortwave cloud radiative effect:

<img width="764" alt="Screen Shot 2024-07-16 at 5 19 01 PM" src="https://github.com/user-attachments/assets/e20aab64-9bfc-4857-bcec-f31c4d5043d0">

Aerosols are increasing, so it's not that something has gone wrong with the rising aerosol forcing:

<img width="743" alt="Screen Shot 2024-07-16 at 5 29 31 PM" src="https://github.com/user-attachments/assets/336ca9d5-fbbf-4873-a896-9f65667ce9c6">

I noted in the CESM project meeting that the AODVIS in 098b was almost double that in LENS2 and we had some discussion about whether the different background aerosol state might be affecting the response to a further rise.  But much of that difference in AODVIS seems to have come in between 092 and 098b, so I think we can rule out an importance of the background aerosols on their subsequent change since 092 and 098b are similar.  092 seems like it compares better with observed estimates of AODVIS e.g., [this paper ](https://www.sciencedirect.com/science/article/pii/S1352231014003252#:~:text=The%20global%20mean%20AOD%20is,Levinson%20and%20Lawrimore%2C%202009) says global mean AOD from 2003-2012 is 0.126

<img width="735" alt="Screen Shot 2024-07-16 at 5 31 24 PM" src="https://github.com/user-attachments/assets/36672dab-4a91-4940-b0ac-4e439eea55f8">

It seems to be dust and sea salt that has changed the most between 092 and 098b.

<img width="1267" alt="Screen Shot 2024-07-16 at 5 32 11 PM" src="https://github.com/user-attachments/assets/33aa2777-17ca-4e92-b7fd-5dab8e5ec940">

**Comment by islasimpson** [comment]
098b and 092 compare fairly well with LENS in the time evolution of the clear sky net shortwave downward at the surface but they differ in the all sky.

<img width="884" alt="Screen Shot 2024-07-16 at 6 11 37 PM" src="https://github.com/user-attachments/assets/b41e96ed-da59-457c-8c50-b9fd989a59c1">

There's a pretty big difference in how the cloud liquid water is behaving.  It increases a lot more over time in LENS2 than it does in the new runs.  So it seems we need to understanding why the cloud liquid is not responding much to aerosols.

<img width="908" alt="Screen Shot 2024-07-16 at 6 13 44 PM" src="https://github.com/user-attachments/assets/985a747f-a723-423d-af8d-1a4862e2dea2">

**Reply by islasimpson** [reply]
Both cloud ice and cloud liquid differ a lot in the climatology too

<img width="869" alt="Screen Shot 2024-07-16 at 6 16 51 PM" src="https://github.com/user-attachments/assets/7844fadb-0b9a-4f23-a592-4a15cd5a4a28">

**Reply by islasimpson** [reply]
p.s. ignore the W/m2 labels in the TGCLDIWP/LWP plots. That's an error

**Comment by swrneale** [comment]
Does it seem like a lack of cloud water sensitivity through the period then? Maybe precipitation efficiency does not decrease in response to aerosols (smaller cloud drops) in CESM3 and it does in CESM2??

**Reply by islasimpson** [reply]
Are ACTNL and ACTNI and CDNUMC the relevant quantities to investigate that?

**Comment by swrneale** [comment]
Also maybe 
micro_pumas_cam.F90:   call addfld('VPRCO' ,       horiz_only,   'A', 'kg/kg/s',  'Vertical average of autoconversion rate'                           )
micro_pumas_cam.F90:   call addfld('RACAU' ,       horiz_only,   'A', 'kg/kg/s',  'Accretion/autoconversion ratio from vertical average'              )

And
What does actual CLDLOW/CLDMED/CLDHGH do through the period?

**Comment by islasimpson** [comment]
Following suggestions from @brianpm in the project meeting yesterday, here's a look at cloud liquid water path versus cloud droplet number concentration.  But first, here's the cloud droplet number concentration and the anomalies which shows that our new runs start off with a higher cloud droplet number concentration and it increases less over time.

<img width="841" alt="Screen Shot 2024-07-17 at 3 59 19 PM" src="https://github.com/user-attachments/assets/7cae3b86-ac68-46ee-9483-dd29591dc8d1">

<img width="830" alt="Screen Shot 2024-07-17 at 3 59 35 PM" src="https://github.com/user-attachments/assets/57b15257-573f-4f6b-ae87-10f4c760c0ba">

Looking at high cloud liquid water varies with CDNUMC, we can also see that the cloud liquid water increases less with cloud number concentration than it did in CESM2

<img width="666" alt="Screen Shot 2024-07-17 at 4 00 13 PM" src="https://github.com/user-attachments/assets/ab0c0344-7d22-437e-a284-bf28a3b95475">

The plot below shows on the left the actual TGCLDLWP and then on the top right estimated TGCLDLWP assuming a linear relationship with CDNUMC where b is the slope of that linear relationship.  That linear assumption works pretty well to explain the different behaviors.

<img width="981" alt="Screen Shot 2024-07-17 at 4 25 04 PM" src="https://github.com/user-attachments/assets/89b551f9-b704-4db5-bacb-65bff9afbb17">

This final plot now focusses on 098b and shows the difference in TGCLDLWP between LENS2 and 098b (black).  It then shows the difference that you'd get if you calculate the 098b value assuming it has the same CDNUMC as LENS2 but its own d TGCLDLWP/dCDNUMC slope (purple) i.e. the effect of the differing slopes alone and then it shows the opposite, where you assume that 098b has the same clope as LENS2 but it has it's differing CDNUMC (green).  Both are contributing to the overall difference in TGCLDLWP with the difference in the rise in CDNUMC being a bit larger than the effect of the differing slopes.

<img width="447" alt="Screen Shot 2024-07-17 at 4 28 57 PM" src="https://github.com/user-attachments/assets/8c84e2d4-2395-4d13-abda-a8f76fa96820">

**Reply by adamrher** [reply]
Since there is a break in slope for large values of CDNUMC in the LENS data, and the 90 series runs are primarily in the large value regime in CDNUMC, I think recomputing the slope b using the values -- not the anomalies -- may yield a smaller b effect?

**Comment by islasimpson** [comment]
Also, just to add some information on the potential role for spin-up issues.  Here is a plot showing global mean temperature for 098b which is the run where the piControl extends for the duration of the run.  The solid line shows the anomalies relative to the average over the full duration of the piControl.  The dashed line shows the anomalies relative to the 10-year running means of the piControl that correspond to the same time after initialization as the historical i.e., any drift in the piControl after the year of initialization is being subtracted out from the historical values.  This suggests that the drift is not the primary issue, unless there's a state dependence of the climate change signal and not simply some drift being added on to the climate change signal.
<img width="757" alt="Screen Shot 2024-07-22 at 12 54 46 PM" src="https://github.com/user-attachments/assets/20965bfd-5e7b-4be6-a363-7ca4b7d49a6b">

**Reply by jfasullo** [reply]
![Screenshot 2024-07-23 at 11 57 21 AM](https://github.com/user-attachments/assets/8dd005aa-d7a9-46ff-a9e9-6dc9a191d1fa)
So from what I see the warming by 1940 is about 0.3 when the contemporaneous drift is accounted for (if one averages through the remaining variability). From what I see from BEST the warming is 0.25 to 0.3 (also averaging through the variability).

**Reply by islasimpson** [reply]
Here's my comparison with BEST.  These are normalized to have zero anomalies from 1850 to 1890.

<img width="757" alt="Screen Shot 2024-07-23 at 12 35 17 PM" src="https://github.com/user-attachments/assets/498a61ee-fc3d-4164-8a1a-4c49c78d4df8">

**Reply by jfasullo** [reply]
Wow - that's closer than I thought it would be.

**Comment by islasimpson** [comment]
As suggested by @adamrher, the slopes of TGCLDLWP versus CDNUMC are not that different if we restrict LENS2 to only the period where CDNUMC is higher.  Indeed, that's the case.  So our high CDNUMC in the new runs is probably the ultimate driver of the difference in the slopes as opposed to a difference in the physics.

<img width="695" alt="Screen Shot 2024-07-23 at 10 55 36 AM" src="https://github.com/user-attachments/assets/86881124-e703-482a-9673-ee3ee8491597">

**Reply by jfasullo** [reply]
As we refine the aerosol burdens, this plot may be useful. The lower left shows the TOA net SW bias relative to EBAF42. The Southern Ocean stood out as being too bright in CESM2 and continues that way in CESM3. Would love to see albedo reduction from  40-60 S. Does sea salt contribute to this or is it something else?
![FSNT mean ann](https://github.com/user-attachments/assets/b7267523-a8d5-4622-93fd-5ed7eccbb68f)

**Comment by cecilehannay** [comment]
The difference between 92 and 98 are:
- Turning on RRTMGP
- the clubb bugfix.
However, we don't have separate simulations for the BLTHIST to distinguish the respective contribution of RRTMGP and clubb bugfix to the changes in AODVIS and in burden.  

For the BLT1850, we have:
92 = baseline
93 = baseline + rrtmgp
97 = baseline +  clubb bugfix
98 = baseline +  clubb bugfix + rrtmgp

I examined these BLT1850 runs to distinguish the respective contribution of RRTMGP and clubb bugfix to the changes in AODVIS and in burden. (Note that I don't have the burdens for 92). 

It seems that the differences are coming from RRTMGP. 

![Screen Shot 2024-07-23 at 3 13 41 PM](https://github.com/user-attachments/assets/a80274cf-701c-4ccd-9d65-69fd8d84dad3)

I checked the namelists, and there are no major changes in tuning parameters or files (except those needed for RRTMGP and also **the file used for `mam4_mode3` in `rad_climate` is different**) 

Th difference in namelists are:
```
radiation_scheme		= 'rrtmg'
solar_htng_spctrl_scl		= .true.
 rad_climate	= .....	'M:mam4_mode3:/glade/campaign/cesm/cesmdata/inputdata/atm/cam/physprops/mam4_mode3_rrtmg_aeronetdust_c141106.nc'
```
```
radiation_scheme		= 'rrtmgp'        
rad_climate	= .....	'M:mam4_mode3:/glade/campaign/cesm/cesmdata/inputdata/atm/cam/physprops/mam4_mode3_rrtmg_aeronetdust_sig1.2_dgnl.40_c150219.nc', 
rrtmgp_coefs_lw_file		= '/glade/work/hannay/cesm_tags/cesm2_3_alpha17f/components/cam/src/physics/rrtmgp/data/rrtmgp-gas-lw-g128.nc'
rrtmgp_coefs_sw_file		= '/glade/work/hannay/cesm_tags/cesm2_3_alpha17f/components/cam/src/physics/rrtmgp/data/rrtmgp-gas-sw-g112.nc'
```

**Reply by cecilehannay** [reply]
I am looking at the physprops files for `mam4_mode3` in ``/glade/campaign/cesm/cesmdata/inputdata/atm/cam/physprops/``
for 
- RRTMG:  `mam4_mode3_rrtmg_aeronetdust_c141106.nc`
- RRTMGP: `mam4_mode3_rrtmg_aeronetdust_sig1.2_dgnl.40_c150219.nc`

and my feeling is there is a bug in the `namelist_defaults_cam.xml`

- **For RRMTG**
we select the `mam4_mode3_rrtmg_aeronetdust_sig1.2_dgnl.40_c150219.nc` for `cam6`:
```
<mam4_mode1_file rad="rrtmg">atm/cam/physprops/mam4_mode1_rrtmg_aeronetdust_c141106.nc</mam4_mode1_file>
<mam4_mode2_file rad="rrtmg">atm/cam/physprops/mam4_mode2_rrtmg_aitkendust_c141106.nc</mam4_mode2_file>
<mam4_mode3_file rad="rrtmg">atm/cam/physprops/mam4_mode3_rrtmg_aeronetdust_c141106.nc</mam4_mode3_file>
<mam4_mode4_file rad="rrtmg">atm/cam/physprops/mam4_mode4_rrtmg_c130628.nc</mam4_mode4_file>

<mam4_mode1_file rad="rrtmg" ver="strat">atm/cam/physprops/mam4_mode1_rrtmg_aeronetdust_sig1.6_dgnh.48_c140304.nc</mam4_mode1_file>
<mam4_mode3_file rad="rrtmg" ver="strat" phys="cam6">atm/cam/physprops/mam4_mode3_rrtmg_aeronetdust_sig1.2_dgnl.40_c150219.nc</mam4_mode3_file>
```
- **For RRMTGP**:
the `cam6` has been removed so we select `mam4_mode3_rrtmg_aeronetdust_sig1.2_dgnl.40_c150219.nc` for cam_dev or cam7:
```
<mam4_mode1_file rad="rrtmgp">atm/cam/physprops/mam4_mode1_rrtmg_aeronetdust_c141106.nc</mam4_mode1_file>
<mam4_mode2_file rad="rrtmgp">atm/cam/physprops/mam4_mode2_rrtmg_aitkendust_c141106.nc</mam4_mode2_file>
<mam4_mode3_file rad="rrtmgp">atm/cam/physprops/mam4_mode3_rrtmg_aeronetdust_c141106.nc</mam4_mode3_file>
<mam4_mode4_file rad="rrtmgp">atm/cam/physprops/mam4_mode4_rrtmg_c130628.nc</mam4_mode4_file

<mam4_mode1_file rad="rrtmgp" ver="strat">atm/cam/physprops/mam4_mode1_rrtmg_aeronetdust_sig1.6_dgnh.48_c140304.nc</mam4_mode1_file>
<mam4_mode3_file rad="rrtmgp" ver="strat">atm/cam/physprops/mam4_mode3_rrtmg_aeronetdust_sig1.2_dgnl.40_c150219.nc</mam4_mode3_file>

```

I don't know which one is correct but these set of lines seem to be a copy/paste and I don't see why `cam6` appears in one case and not the other.

**Reply by cecilehannay** [reply]
@fvitt: do you know anything about the mam4_mode3_file in the physprop and whether it should be set to  
- `mam4_mode3_rrtmg_aeronetdust_c141106.nc` or 
- `mam4_mode3_rrtmg_aeronetdust_sig1.2_dgnl.40_c150219.nc`

**Reply by fvitt** [reply]
For MAM4 in cam7 we reverted the size parameters for the coarse mode (mode3) back to what was used in MAM3 (in cam5), as of CAM tag cam6_3_087.  So I believe mam4_mode3_rrtmg_aeronetdust_c141106.nc is the correct file.   

My understanding is that the coarse mode in cam6 included dust and stratospheric aerosols and the size params did not represent dust very well.  Now we have MAM5 where the 5th mode is for stratospheric aerosols for configurations that include stratospheric aerosols.

@tilmes or @lkemmons please weigh in if I am mistaken.

**Reply by tilmes** [reply]
That is correct, we want mam4_mode3_rrtmg_aeronetdust_c141106.nc for coarse mode for CAM and having the wrong file can make a large differences in the dust burden and seasalt as well.
Simone

**Reply by tilmes** [reply]
@cecilehannay could you clarify which of the cases had the wrong file, was it 098b or 092?

**Reply by cecilehannay** [reply]
It is 98b that used the "wrong" file. 
- 92: RRTMG: mam4_mode3_rrtmg_aeronetdust_c141106.nc
- 98b: RRTMGP: mam4_mode3_rrtmg_aeronetdust_sig1.2_dgnl.40_c150219.nc

@tilmes and @fvitt: Thanks for the clarifications. I should update the namelist default to fix the file when RRTMGP is on. I will open a request.

**Comment by islasimpson** [comment]
Here is a plot of the burdens in the latest runs.  Our burden of sea salt with the new tuning in 102 is now a lot lower than it was in LENS2.  We probably need to be comparing with observations to find out what's reasonable.  The reverting to the old DMS scheme in 101 brings SO4 down but not all the way down to what it was in LENS2.

<img width="1276" alt="Screen Shot 2024-07-30 at 11 26 15 AM" src="https://github.com/user-attachments/assets/462e0b4b-5651-4558-89ab-54a28ea13525">
